### ISSUE TITLE
mount userspace (space execute --mnt) does not work

### ISSUE THREAD

**Issue by fpemud** [Friz-zy/pyspaces#6]
```
mysys / # mount | grep /mnt
mysys / # python2.7 /usr/bin/space execute --mnt bash -c 'mount -t tmpfs tmpfs /mnt'
mysys / # mount | grep /mnt
tmpfs on /mnt type tmpfs (rw,relatime)
```

From the above experiment, we can see that the /mnt mount is not isolated in the "mount namespace" of the new process.

unshare behaves correctly on my system:

```
mysys / # mount | grep /mnt
mysys / # unshare -m bash -c 'mount -t tmpfs tmpfs /mnt'
mysys / # mount | grep /mnt
```

It is very ugly to call "unshare" in my python script compared to the pyspaces.Container API.
Although the experiment is done with CLI, but I know the API behaves the same.

**Comment by fpemud** [comment]
I found there's no namespaces manipulating code in Container class?!
It seems only Inject class is usable...

Ah...the README says "mount" function is not available.

**Comment by Friz-zy** [comment]
No, all magic is under Container class and [it occurs at time of Clone syscall](http://man7.org/linux/man-pages/man7/namespaces.7.html)
